### ISSUE TITLE
Refactoring `simplify-fstring-formatting` replaces newlines with `\x1f` in multiline strings

### ISSUE THREAD

**Issue by ruancomelli** [sourcery-ai/sourcery#240]
### Issue description or question

The `simplify-fstring-formatting` refactoring is replacing newlines with the `\x1f` character in multiline strings. For instance,
```python
def ooops():
    return f"""
        Sourcery is
        {f"{'awesome'.upper()}"}
    """
```
is refactored as
```python
def ooops():
    return f"""\x1f        Sourcery is\x1f        {'awesome'.upper()}\x1f    """
```

The expected result would be
```python
def ooops():
    return f"""
        Sourcery is
        {'awesome'.upper()}
    """
```

### Sourcery Version

v0.11.5

### Code editor or IDE name and version

VS Code 1.67.2

### OS name and version

Ubuntu 20.04.4 LTS

**Comment by bm424** [comment]
Duplicate of https://github.com/sourcery-ai/core/issues/1492

**Comment by Hellebore** [comment]
Addressed with release of 0.12.1
